eos-payg-ctl/eos-payg-ctl

```diff
@@ -40,15 +40,29 @@ DBUS_PROPERTIES_INTERFACE = "org.freedesktop.DBus.Properties"
 CLOCK_BOOTTIME = getattr(time, 'CLOCK_BOOTTIME', 7)
 
 
-def __format_boottime_timestamp(boottime_timestamp):
-    # Convert from CLOCK_BOOTTIME to seconds since the epoch
-    timestamp_delta = boottime_timestamp - time.clock_gettime(CLOCK_BOOTTIME)
-    utc_timestamp = time.time() + timestamp_delta
-
-    # Format that as a human-readable local time
-    expiry_utc = dt.datetime.fromtimestamp(utc_timestamp, tz=tz.tzutc())
-    expiry_local = expiry_utc.astimezone(tz.tzlocal())
-    return str(expiry_local)
+def __make_boottime_formatter():
```

Comment:
Personally I think it would be more readable to have `now_boottime` and `now_utc` initialize statically rather than using a function to get a function, but it's not worth holding up the merge over this.

Reply:
That's fair. I really like higher-order functions, and what I like about this approach is that `now_boottime` and `now_utc` don't leak out into global scope, but perhaps you're right that it makes it less readable for little benefit.